src/os/ui/datetime/datecontrol.js

```diff
@@ -165,6 +174,14 @@ os.ui.datetime.DateControlCtrl.prototype.onStartDateChanged_ = function(newVal,
 
     goog.log.fine(os.ui.datetime.DateControlCtrl.LOGGER_,
         'start changed: ' + this['startDate'].toUTCString() + ' to ' + this['endDate'].toUTCString());
+
+    // open the datepicker for the end date
+    const endDatePicker = $('#timelineEndDate');
+    if (endDatePicker && this['showEndDateCalendar']) {
```

Comment:
This flag is a bit fragile because the start date may be changed outside of this directive. For example, if you open the timeline and change the yellow load range the end date picker will open.

Instead of using a flag to determine when the end date should open, find a way to notify this controller when the user specifically selects a date in the start date picker.